proto/monitor_service.proto

```diff
@@ -11,6 +11,16 @@ message StackTraceResponse {
   map<string, string> rpc_traces = 2;
 }
 
+message ProfilingRequest {
+  // How long the profiling should last.
+  uint64 time = 1;
+}
```

Comment:
Perhaps we will add more kinds of profiling (such as call graphs) in the future, for example, see the options of [flamegraph-rs](https://github.com/flamegraph-rs/flamegraph). But this worry because this doesn't need to be backward compatible necessarily.

Reply:
Could you please specify the unit in the field name? Like `time_ms`.